float32 value
---
string message